modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -742,7 +742,7 @@ describe('User CRUD tests', function () {
               }
 
               // Call the assertion callback
-              userInfoRes.body.message.should.equal('Username already exists');
+              userInfoRes.body.message.toLowerCase().should.containEql('username already exists');
```

Comment:
@trendzetter FYI, I'm using MongoDB `3.0.7` & don't have this issue. So perhaps indicate what exact version, or clarify that some release after `3.0.7` changes the validation messages, in your comment here.